mobify/hijax
# [WIP] Version 0.2.0

Status: **WIP**
Reviewers: @fractaltheory @haroldtreen @jnurse @Helen-Mobify 

Preparing 0.2.0 release with @jnurse and @Helen-Mobify's changes.

## Todos:
- [x] Merge Jen's PR
- [x] Merge Helen's PR
- [x] Update readme and build files
- [ ] Release prep

### Feedback:
_none so far_

## Related PRs
- [x] https://github.com/mobify/hijax/pull/8
- [x] https://github.com/mobify/hijax/pull/11

## How to test
- Checkout this repo
- Ensure tests pass
- Ensure that new features work as expected
- Break features and ensure tests correctly detect failures
